uniffi_bindgen/src/interface/mod.rs

```diff
@@ -219,6 +219,25 @@ impl ComponentInterface {
         self.errors.iter().find(|e| e.name == name)
     }
 
+    /// Should we generate read/lower functions for errors?
```

Comment:
This lower/read is from the POV of the foreign bindings, right? Maybe change this to start with "Should foreign bindings generate ..."?

Reply:
oh, no, it's for the scaffolding - so should "lower" be "lift"?